media/css/main.css

```diff
@@ -10,13 +10,75 @@ body {
 body > .container {
   background-color: #fff;
   min-height: 100%;
-  padding-top: 75px;
+  padding-top: 80px;
+}
+
+img {
+    width: 100%;
```

Comment:
This seems too general, and has the potential to break the layout in unexpected ways for images that aren't meant to fill the width, like logos. I suggest using `class="img-fluid"` explicitly on the images you want to widen https://getbootstrap.com/docs/4.0/content/images/#responsive-images

Reply:
Good point.  I removed it and added the class name where its needed.